src/analytics/Averages.js

```diff
@@ -0,0 +1,199 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import { Col, Row, Table, InputGroup, InputGroupAddon, InputGroupText, CustomInput, NavLink } from 'reactstrap';
+
+import { getCmc } from 'utils/Card';
+import { sortIntoGroups, getSorts } from 'utils/Sort';
+import ErrorBoundary from 'components/ErrorBoundary';
+
+const useSortableData = (items, config = null) => {
```

Comment:
This should be moved to common file it can be imported from instead of duplicated.